### ISSUE TITLE
Bazel 0.25.0 compatibility

### ISSUE THREAD

**Issue by Kernald** [grailbio/bazel-toolchain#12]
Trying to use those rules with Bazel 0.25.0 causes an issue, about `conditional_cc_toolchain` not being contained by `//toolchain:rules.bzl`. According to the [changelog](https://github.com/bazelbuild/bazel/releases/tag/0.25.0), `--incompatible_no_transitive_loads` is now enabled by default, which is what causes the issue, as [`toolchain/rules.bzl` is indeed loading this symbol from `toolchain/internal/configure.bzl`](https://github.com/grailbio/bazel-toolchain/blob/master/toolchain/rules.bzl#L17). I guess the solution would be to define a `conditional_cc_toolchain` symbol in `toolchain/rules.bzl` just wrapping the calls to the internal definition.

There's also a second flag that has been flipped, causing issue with this toolchain: `--incompatible_remove_cpu_and_compiler_attributes_from_cc_toolchain`. Explanations [here](https://github.com/bazelbuild/bazel/issues/7075), but the tl;dr is: those attributes were previously mandatory but unused, we can safely remove them entirely.

And finally, a third one: `--incompatible_disable_crosstool_file`. The [tracking bug](https://github.com/bazelbuild/bazel/issues/7320) explains how to use a migration script to convert the crosstool file in a Starlark structure.

**Comment by siddharthab** [comment]
Thank you so much for the detailed report.

I have been able to take care of the first two items, but migrating the CROSSTOOLS file is a somewhat bigger task. I can get to it by this Saturday. I was able to run the migration tools to get a preview of the changes.

I should be better at being ready for breaking changes. Maybe I will start using bazelisk and enabling all incompatible flags in my Travis tests.

Thanks again.
